Makefile

```diff
@@ -242,9 +242,12 @@ eunit-fortuna: internal-build
 
 all-tests: eunit ct
 
-docker:
+docker: dockerignore-check
 	@docker build -t aeternity/aeternity:local .
 
+dockerignore-check: | .gitignore .dockerignore
```

Comment:
Why not simply use `.dockerignore: .gitignore` ? That way the target wouldn't be PHONY.

Reply:
I removed the fix, I applied this change.

```
$ make .dockerignore 
make: `.dockerignore' is up to date.
```
exit status zero.

So I plan not to apply this change.

Reply:
Right. My comment didn't really make sense in the target context.